frontend/apps/pwa/src/features/former/FormName.tsx

```diff
@@ -17,7 +17,7 @@ export const FormName: FC<FormNameProps> = (props) => {
           className="form-control"
           id="formName"
           type="text"
-          value={formName || ''}
+          value={formName ?? ''}
```

Comment:
would we not want the default value if formName is ANY falsy value? or do we want the default only if formName is null or undefined? I'd have kept ||, what's your argument here?

Reply:
I guess in this case they are equivalent since they resolve to `''` . Gonna switch it back anyway